code/modules/SCP/SCPs/SCP-049.dm

```diff
@@ -75,9 +75,9 @@ GLOBAL_LIST_EMPTY(scp049_1s)
 	skillset.skill_list[SKILL_COOKING] = SKILL_TRAINED
 	skillset.skill_list[SKILL_BOTANY] = SKILL_TRAINED
 	skillset.skill_list[SKILL_HAULING] = SKILL_MASTER
-	skillset.skill_list[SKILL_COMBAT] = SKILL_EXPERIENCED
+	skillset.skill_list[SKILL_COMBAT] = SKILL_MASTER
```

Comment:
This lets 049 counter any grab by being on harm intent, so no. It stays at experienced.

Reply:
damn

Reply:
> This lets 049 counter any grab by being on harm intent, so no. It stays at experienced.

Actually, this might be wrong, I'm checking the code rn.

Reply:
nevermind found it
`var/skill_mod = Clamp(affecting.get_skill_difference(SKILL_COMBAT, assailant), -1, 1)`
master makes the mod so high you can resist grabs near instantly, which is bad for 049 because it would mean containing him relies solely on him agreeing to be contained.